tools/boxstarter/node_boxstarter_vscode

```diff
@@ -0,0 +1,25 @@
+# Boxstarter (http://boxstarter.org/) script for Node.js prerequisites,
+#  including Visual Studio Code
+#
+# To install either open this link in IE or Edge:
+#  http://boxstarter.org/package/nr/url?https://raw.githubusercontent.com/nodejs/node/master/tools/boxstarter/node_boxstarter_vscode
+#
+# Or run those commands in a Powershell terminal:
+#  iex ((New-Object System.Net.WebClient).DownloadString('http://boxstarter.org/bootstrapper.ps1'))
+#  get-boxstarter -Force
+#  Install-BoxstarterPackage https://raw.githubusercontent.com/nodejs/node/master/tools/boxstarter/node_boxstarter_vscode -DisableReboots
+#
+# For more detail see
+# https://github.com/nodejs/node/blob/master/tools/boxstarter/README.md
+#
+
+# Git and Unix tools will be added to the PATH
+choco install git -params /GitAndUnixToolsOnPath -y
```

Comment:
I'd prefer it if we didn't install the editor as well - most people being onboarded have their own preference for an editor.